src/app/containers/StoryPromo/MediaIndicator/index.jsx

```diff
@@ -1,49 +1,71 @@
 import React from 'react';
 import moment from 'moment-timezone';
-import { shape, bool, string } from 'prop-types';
+import { shape, bool, string, oneOfType } from 'prop-types';
 import MediaIndicatorComp from '@bbc/psammead-media-indicator';
-import pathOr from 'ramda/src/pathOr';
-import pick from 'ramda/src/pick';
-import { storyItem } from '../../../models/propTypes/storyItem';
+import path from 'ramda/src/path';
+import { storyItem, linkPromo } from '../../../models/propTypes/storyItem';
 import formatDuration from '../../../lib/utilities/formatDuration';
 
-const MediaIndicator = ({ item, topStory, service, indexAlsos }) => {
-  const isPGL = pathOr(null, ['cpsType'], item) === 'PGL';
-  const isMedia = pathOr(null, ['cpsType'], item) === 'MAP';
-  const hasMediaInfo = pathOr(null, ['media'], item);
+const getAssetContentTypes = item => {
+  const mediaContentTypes = ['audio'];
+  const type = path(['contentType'], item) && item.contentType.toLowerCase();
+  return mediaContentTypes.includes(type) ? type : null;
+};
+
+const getCpsMediaTypes = item => {
+  const isPGL = path(['cpsType'], item) === 'PGL';
+  const isCpsMedia = path(['cpsType'], item) === 'MAP';
+  const hasMediaInfo = path(['media'], item);
 
   // Only build a media indicator if this is a photo gallery or media item
-  if (!isPGL && (!isMedia || !hasMediaInfo)) {
+  if (!isPGL && (!isCpsMedia || !hasMediaInfo)) {
     return null;
   }
+  const type = isPGL ? 'photogallery' : path(['media', 'format'], item);
+  return type || null;
+};
 
-  const type = isPGL ? 'photogallery' : pathOr(null, ['media', 'format'], item);
+const getMediaType = item => {
+  const isAssetTypeMedia = path(['assetTypeCode'], item);
+  return isAssetTypeMedia ? getAssetContentTypes(item) : getCpsMediaTypes(item);
+};
+
+const MediaIndicator = ({ item, topStory, service, indexAlsos }) => {
+  const type = getMediaType(item);
```

Comment:
This could return null. We can add a null check here 
```js
if(!type) {
  return null
}
```
This will also mean that we don't have the `type &&` as we do below.